src/VCF_Tasks.cpp

```diff
@@ -18,11 +22,11 @@ bool run_read_adc1_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo&
     adc_1.sample(); // Samples all eight channels.
     adc_1.convert(); // Converts all eight channels.
```

Comment:
this won't build, this needs the ADC fix. That should happen on another branch ASAP and then we'll merge it into here.